docs/v2_migration.md

```diff
@@ -55,6 +55,31 @@ When different build tools are use, at least one layout needs to be set.
 The `src_folder` must be the same when using different layouts and should
 not depend on settings or options.
 
+## CMakeToolchain
+
+The old `CMake.definition` should be replaced by `CMakeToolchain.variables` and moved to the `generate` method.
+However, certain options need to be passed as `cache_variables`. You'll need to check project's `CMakeLists.txt`
```

Comment:
should I check just project's **root** `CMakeLists.txt`, or all of them?

Reply:
Depends on the project... I mean maybe if they have multiple layers with different options and multiple projects all being combined yeah you might possibly need to -- but I have not see that thankfully